public/app/core/components/ColorScale/ColorScale.tsx

```diff
@@ -64,15 +64,15 @@ export const ColorScale = ({ colorPalette, min, max, display, hoverValue, useSto
       </div>
       {display && (
         <div className={styles.followerContainer}>
+          <div className={styles.legendValues}>
+            <span>{display(min)}</span>
+            <span>{display(max)}</span>
+          </div>
           {percent != null && (scaleHover.isShown || hoverValue !== undefined) && (
             <span className={styles.hoverValue} style={{ left: `${percent - LEFT_OFFSET}%` }}>
```

Comment:
can this just be a static `transformX(-50%)` defined in css?

Reply:
👍